packages/yoga/src/IconWrapper/IconWrapper.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { oneOf, elementType, string } from 'prop-types';
+import { withTheme } from 'styled-components';
+
+const IconWrapper = ({
+  as: Component,
+  width,
+  height,
+  fill,
+  stroke,
+  theme,
+  ...props
+}) => (
+  <Component
+    width={theme.yoga.spacing[width] || width}
+    height={theme.yoga.spacing[height] || height}
+    {...(fill && { fill: theme.yoga.colors[fill] || fill })}
+    {...(stroke && { stroke: theme.yoga.colors[stroke] || stroke })}
+    {...props}
+  />
+);
+
+const commonSizes = [
+  'xxxsmall',
+  'xxsmall',
+  'xsmall',
+  'small',
+  'medium',
+  'large',
+  'xlarge',
+  'xxlarge',
+  'xxxlarge',
+  'huge',
+  'xhuge',
+];
+
+IconWrapper.propTypes = {
+  as: elementType.isRequired,
+  fill: string,
+  stroke: string,
+  width: oneOf(commonSizes),
+  height: oneOf(commonSizes),
+};
+
+IconWrapper.defaultProps = {
```

Comment:
Maybe we don't need `defaultProps`, since the `svg` itself already has it. 🤔

Reply:
Why would we remove it? We're not only sending directly values to svg and to remove `defaultProps` we would need to make fill, stroke and etc required properties rather than letting a common default.

Reply:
Ah, you are right, we need default props if the prop is not required. Sorry. 🤦🏼‍♂️ 

I think what I meant is: Why can't all these default values be `undefined` since the `svg` itself already has the default values.